### ISSUE TITLE
TSDZ2 250w 36v....

### ISSUE THREAD

**Issue by KILLERD0G** [emmebrusa/TSDZ2-Smart-EBike-1#113]
Just wondering when i try the beta version and i use throttle and pull the brakes hard, why does the throttle not enage again. Once the brakes are pulled on the motor goes really slow and struggle to start again, until i turn the bike off and then back on. Any ideas ? My settings are below....
![Advanced](https://github.com/user-attachments/assets/fe3f179e-865d-4568-b62b-82b6a6f0efdc)
![Assist](https://github.com/user-attachments/assets/27e76627-0396-4fa6-9aac-ff66f51543f9)
![Basic](https://github.com/user-attachments/assets/5b1997c5-dd05-481c-a288-05d56e21a020)

**Comment by KILLERD0G** [comment]
Forgot to add my brakes are hydralic, i am not using any brakes sensors....

**Comment by pepsonEL** [comment]
Hi
Did you resolvced this problem ?

**Comment by KILLERD0G** [comment]
Yes and no. I went back to the april relase and stopped using the beta's... I have changed my settings now and i am having some good results.
